Library/Homebrew/download_strategy.rb

```diff
@@ -282,6 +283,14 @@ def initialize(name, resource)
   end
 
   def fetch
+    if ENV["HOMEBREW_ARTIFACT_DOMAIN"]
+      if @url.start_with?(BottleSpecification::DEFAULT_DOMAIN)
+        @url.sub!(BottleSpecification::DEFAULT_DOMAIN, ENV["HOMEBREW_ARTIFACT_DOMAIN"])
```

Comment:
So `HOMEBREW_ARTIFACT_DOMAIN` will overwrite `HOMEBREW_BOTTLE_DOMAIN`. And any reason bottles should be treated differently?